client/lib/track-form/index.jsx

```diff
@@ -22,9 +22,10 @@ export const trackForm = WrappedComponent => class TrackFormComponent extends Co
 		this.setState( newState, callback );
 	};
 
-	replaceFields = ( fields, callback ) => {
+	replaceFields = ( fields, callback, keepOldFields = true ) => {
```

Comment:
> This signature feels a little bit unfortunate, the callback in the middle doesn't follow convention, but I don't see a clean way around it.

I totally agree, but let's leave that for another time.

> I have a preference for activating this option, by that I mean inverting the boolean clearFields = false (or flushFields = false or similar) by default, and you activate it by setting it to true. Just a sensation I have reading this which you should feel free to ignore.

The default behavior so far was to have it enabled, which is why I'm introducing an option to disable it. The fact is we have two usages: one keeps the old fields, the other doesn't. So it's either this way or the other one. But let's keep it like this for now, and we can always invert it without pain.